crates/edr_op/src/hardfork/generated.rs

```diff
@@ -96,7 +96,7 @@ pub mod xterio_eth;
 pub mod zora;
 
 pub(super) fn chain_configs() -> HashMap<u64, ChainConfig<Hardfork>> {
-    HashMap::from([
+    [
```

Comment:
The `HashMap::from` implementation was not generic over the hasher's generic type. As such, we need can no longer use this function, after the adoption of alloy's exported `HashMap` type.